src/io/epanet.jl

```diff
@@ -404,7 +404,19 @@ function _string_time_to_seconds(s::AbstractString)
         end
     end
 end
-    
+
+function _clean_nothing!(data)
+    for (key, value) in data
+        if isa(value, Dict)
+            value = _clean_nothing!(value)
+        elseif value == nothing
+            delete!(data, key)
+        end
+    end
+
+    return data
```

Comment:
When a function mutates an argument I would not have it return a value.  I save that pattern for cases where when the function is non-distructive.

Reply:
Fixed in 40767a3db018d629bf3cac7b1e0739116094cc8d.